book/internals/adders/adders_with_properties.nim

```diff
@@ -0,0 +1,70 @@
+import nimib, nimibook
+
+nbInit(theme = useNimibook)
+nbText: """
+## **Adders with properties**
+Let's make a custom widget that stores Widgets in a `Table[string, Widget]` and displays the widget next to the key it was stored with.
+
+First we need to modify our adder, telling it that there will be additional parameters. 
+
+```nim
+...
+viewable CustomBox:
+  myChildren: Table[string, Widget] # The child-widget field
+
+  adder add {.key: none(string).}: 
+...
+``` 
```

Comment:
The closing ` ``` ` are shown on the website for some reason, maybe a nimib bug?

Reply:
Yeah that's a nimibook bug. As is, that the rest of the nbText block will be treated as part of the code block.
I workarounded that for now, by leaving the backticks out. That renders properly for now, and by closing the nbText block and opening a new one I avoid the rest of the text being displayed as code-block.

Reply:
The issue appears to have been a trailing whitespace. Not sure if that was a nimibook bug or not, the authors have been informed, and I fixed the thing and removed the workaround. :partying_face: